attack_range.conf

```diff
@@ -268,10 +287,19 @@ windows_domain_controller_os = Windows_Server-2016-English-Full-Base-*
 # specify the windows domain controller operating System
 # Currently only Windows_Server_2016 is supported
 
+windows_domain_controller_zeek_capture = 0
+#specify if you want to capture wire data from the Windows Domain Controller
+# possible values: 1, 0
+
 
 [windows_server]
 # customize the windows server
 
+windows_server_instance_type = m5.2xlarge
```

Comment:
This should be its own PR. Same for the ones that follow down

Reply:
I'll change the ones back that don't have any bearing on the capture interfaces.  The ones for the Windows hosts need to be defined as Nitro instances, or else the VPC mirroring won't work.  I've put them into the Zeek config block for users to modify there if that makes more sense?